mailpoet/tests/integration/Cron/Workers/SendingQueue/Tasks/NewsletterTest.php

```
@@ -392,6 +393,47 @@ public function testItDoesNotReplaceSubscriberDataInLinksWhenTrackingIsNotEnable
+      'text' => '[Leave a review]([woocommerce/order-review-url])',
+    ]);
+    $this->sendingQueuesRepository->persist($sendingQueue);
+    $this->sendingQueuesRepository->flush();
+
+    try {
+      $this->newsletterTask->prepareNewsletterForSending(
+        $newsletter,
+        $this->subscriber,
+        $sendingQueue
+      );
+      self::fail('Expected order review URL resolution to stop sending.');
```

PHPStan (2.1.54)

[error] Call to an undefined static method MailPoet\Test\Cron\Workers\SendingQueue\Tasks\NewsletterTest::fail().


(staticMethod.notFound)